packages/visualizations/src/components/MapPoi/Map.ts

```diff
@@ -313,13 +375,10 @@ export default class MapPOI {
         this.queue((map) => map.jumpTo(options));
     }
 
-    setPopupsConfiguration(config: PopupsConfiguration) {
-        this.popupsConfiguration = config;
-        this.queue((map) => {
-            if (this.activeFeature) {
-                this.updatePopup(map, [this.activeFeature]);
-            }
-        });
+    setpopupConfigurationByLayers(config: PopupConfigurationByLayers) {
+        this.popupConfigurationByLayers = config;
+        this.updatePopupContent();
+        this.updatePopupDisplay();
```

Comment:
After our discussions, I understand why we need to update content and display when `PopupConfigurationByLayers` changes but maybe a comment would be useful to understand that `updatePopupContent` and `updatePopupDisplay` use `PopupConfigurationByLayers` so whenever the config changes we must sync it to be sure that the popup has the latest config.

Reply:
Done in https://github.com/opendatasoft/ods-dataviz-sdk/pull/205/commits/618ec9ed9bc7005cacdea6a652b660ec35318e7a